pvpanic/PVPanic Package/pvpanic_pack.props

```diff
@@ -0,0 +1,15 @@
+<!--
+***********************************************************************************************
+pvpanic_pack.props
+Enabling and customizing virtio build features
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="Platform">
+  <PropertyGroup>
+    <Feature_AlwaysDefaultVendor>false</Feature_AlwaysDefaultVendor>
+    <Feature_AdjustInf>true</Feature_AdjustInf>
+    <RHEL_COPYRIGHT_STARTING_YEAR>2015</RHEL_COPYRIGHT_STARTING_YEAR>
```

Comment:
This is unfortunate. Now we have these per-driver .props files with vendor entries in them. Makes it harder to separate vendor-specific properties from the general build properties. Ideally everything vendor specific would stay in its dedicated files.

Just brainstorming here. What if put the copyright starting years (plus anything else that might be different - like for example the balloon service/no-service feature) in per-driver per-vendor files and then:

  <Import Project="pvpanic_pack.$(_VENDOR_).props" />

or

  <Import Project="pvpanic_pack.$(_VENDOR_).props" Condition="exists(pvpanic_pack.$(_VENDOR_).props)" />


Or another option. To avoid adding many new files, this file would have something like:

  <CopyrightYears>%(PVPanicCopyrightYears)</CopyrightYears>

(similar for other drivers)

and PVPanicCopyrightYears would be defined in Driver.RHEL.props and Driver.VZ.props with the right years.

I'll think about it some more but I really don't like the current solution because it breaks the otherwise elegant separation of the vendor-specific props.

Also not super-excited about the for (;;) loop in Driver.Common.targets. I would prefer if the copyright string was concatenated by MSBuild and only one regex-replace pass was needed to massage the .inf files into their final form. Thanks!